### ISSUE TITLE
1.18.10 is missing updated isStepDivisible checks

### ISSUE THREAD

**Issue by rockerBOO** [tajo/react-range#151]
In 703df3c603194a25f6d763f8aa4a90eef5b24f48 there was an updated to fix minor step changes but in the NPM version it is missing.

in `node_modules/react-range/lib/utils.js`
```javascript
function isStepDivisible(min, max, step) {
    var res = (max - min) / step;
    return parseInt(res.toString(), 10) === res;
}
```

Expected:
```javascript
function isStepDivisible(min, max, step) {
  const res = (max - min) / step;
  const precision = 8;
  const roundedRes = Number(res.toFixed(precision));
  return parseInt(roundedRes.toString(), 10) === roundedRes;
}
```

Would love to have this fix. 

Thank you.

**Comment by tajo** [comment]
Should be fixed now with 1.18.1.

**Comment by rockerBOO** [comment]
Perfect ! Thank you!
